FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+# WebView2Find API
+
+## Background
+
+The WebView2Find API offers methods and events for text finding and navigation
+within a WebView2 control. It enables developers to programmatically initiate find
+operations, navigate find results, suppress default UI, and customize find configurations
+like query and find direction. It also tracks the status of operations, indicating
+completion, match count changes, and match index changes.
+
+## Examples
+
+
+#### Description
+
+To initiate a find operation in a WebView2 control, use the `StartFind` method.
```

Comment:
Are we pre-committed to using "Find" as both a noun and a verb in this API, e.g. because of industry standards or existing webview2 code?

"Find" is commonly a verb in API method names.

Here "Find" is will be a noun twice in `coreWebView2.Find.StartFindAsync(...`, and a noun and verb once each in `coreWebView2.Find.FindNext()`.

Reply:
I also find the naming awkward here. I think we should consider something like `FindOperation` or `FindSession` over `Find`.

Reply:
The old API ([Microsoft internal link](https://microsoft.sharepoint.com/:w:/r/teams/OSG_DevX/_layouts/15/Doc.aspx?sourcedoc=%7BFB431E6D-55C4-4088-8AE6-F5923F66CFCC%7D&file=WebPlatform%20Find%20On%20Page.docx&action=default&mobileredirect=true&DefaultItemOpen=1&share=IQFtHkP7xFWIQIrm9ZI_Zs_MAd4xulCXvVszPGX1lg5jenc)) called it a "find session", initiated by a "find request".

Reply:
I think part of the awkwardness is usage of the word "find" rather than "search"; "search" as both a noun and a verb is more common, at least in en-us. (But "find" is the right name for this feature)

Reply:
Please update:
 * CoreWebView2.Find -> CoreWebView2.FindSession
 * StartFindAsync -> StartAsync
 * StopFind -> Stop

(but leave FindNext/Previous as is with Find)